packages/legend-studio-extension-query-builder/src/stores/QueryBuilder_EditorExtensionState.ts

```diff
@@ -95,6 +95,7 @@ export class QueryBuilder_EditorExtensionState extends EditorExtensionState {
     runtime: Runtime,
     onSave: (lambda: RawLambda) => Promise<void>,
     disableCompile: boolean,
+    disableParameters?: boolean | undefined,
```

Comment:
I think we should refactor this a bit in terms of how we pass in config for query builder, refer to my extended comment in `QueryBuilderParamterState`